src/widgets/communication/ui/communication.tsx

```diff
@@ -61,7 +61,7 @@ export const Communication = ({ courseName, lang = 'en' }: RequiredProps) => {
             <DiscordLogo />
           </figure>
           <div>
-            <Subtitle text={localizedContent[lang].subTitle} />
+            <Subtitle>{localizedContent[lang].subTitle}</Subtitle>
```

Comment:
Let's remove these extra styles to make it consistent:

![image](https://github.com/user-attachments/assets/f8470816-7439-46eb-b163-e97d1d5716af)

![image](https://github.com/user-attachments/assets/a34d6306-fd9b-496d-913c-9a3dd4d1cb7c)